cockatrice/src/chatview.h

```diff
@@ -44,6 +44,7 @@ private slots:
     ChatView(const TabSupervisor *_tabSupervisor, TabGame *_game, bool _showTimestamps, QWidget *parent = 0);
     void retranslateUi();
     void appendHtml(const QString &html);
```

Comment:
I'd like to see appendHtml private since otherwise it's going to be easy to add code that accidently calls the wrong thing. HtmlServerMessage can be protected